e2e/testdata/credential-install-overload.golden

```diff
@@ -0,0 +1 @@
+ambiguous credential resolution: "secret1" is already present in base credential sets, cannot merge
```

Comment:
I guess it's hard to backtrack which credential set, once everything is merged?

Reply:
yes.